### DISCUSSION TITLE
What are the invoices?

### DISCUSSION THREAD

**Discussion by amuradyan** [pmuradian/Garcon#2] [Q&A] [answered]
@pmuradian, in the readme, you mention that _Garçon_ should:

> send corresponding invoices to your teammates who placed orders

Can you elaborate more on the invoices? I'd like to know what they may contain, when /during the flow/ are they sent, are they updated, how are they populated, and such.

**Comment by pmuradian** [comment, marked as answer]
5 minutes before the order is made, I want Garçon to post a list of orders in following format
[requester] - [dishes] : [total price in AMD] AMD 
This is a way to be sure that everyones order is in the list.
Requester should also be able to remove/update his order /readme should be updated/

**Reply by amuradyan** [reply]
I see. I am thinking of a tab /as in _list of orders_/ here. 

I propose we open a story /issue/ on _the last call for order checks_, so we can discuss it separately.
